section4/reflection.md

```diff
@@ -1,22 +1,41 @@
 ## Section 4 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
+It felt better in some respects, worse in others. 25 minutes is too small a time for me, as it can take me a while to figure out where I left off.
```

Comment:
Good reflection! Totally ok to "up" that time to something that feels better for you. (BTW I am the same - set my timer for 50 then take a little longer of a break)